src/components/data/LocalVariablesContent.js

```diff
@@ -236,7 +292,9 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
           Utils.withBusyState(setBusy)
         )(async () => {
           setDeleteIndex()
-          await Ajax().Workspaces.workspace(namespace, name).deleteAttributes([amendedAttributes[deleteIndex][0]])
+          await Ajax().Workspaces.workspace(namespace, name)
+            .deleteAttributes([amendedAttributes[deleteIndex][0],
+              toDescriptionKey(amendedAttributes[deleteIndex][0])])
```

Comment:
Any reason for this to be on two lines instead of one? Feel free to ignore if you think it's easier to read this way.
```suggestion
            .deleteAttributes([amendedAttributes[deleteIndex][0], toDescriptionKey(amendedAttributes[deleteIndex][0])])
```

Reply:
Made it a single line.